### PR TITLE
Explain how to run Prometheus locally

### PR DESCRIPTION
**Pull request by MathieuDutSik**
## Motivation

The need for debugging metrics is very important. The executive decision has been to use the Prometheus system

## Proposal

The explanation has been added to the `CONTRIBUTING.md` file. If there is a demand, the insertion
of the scripts to process the results could be done.

## Test Plan

It has worked for me on my laptop.

## Release Plan

Nothing relevant.

## Links

- [reviewer checklist](https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md#reviewer-checklist)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -191,6 +191,43 @@ A simple way to use this on a laptop is the following:
...
 
+### Tracking metrics
+
+The prometheus system is used for keeping track of the validator in a Kubernetes setting.
+However, this can also be used locally to run test case and then get metrics to work on.
+This is a little bit more indirect than having runtimes in log file, but works fine.
+The steps are the following:
+
+1. The list of created metric server can be seen in the validator logs with the entries "Starting to serve metrics".
+2. The list of nodes served if 4 validors are used `is 0.0.0.0:XXXXX` with `XXXXX` being 11000, 11001, 11100, 11101, 11200, 11201, 11300, 11301.
```
Comment (ma2bd) [inline CONTRIBUTING.md:202]: typo: validators

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -191,6 +191,43 @@ A simple way to use this on a laptop is the following:
...
 See the documentation of `tokio-console` for more details.
 
+### Tracking metrics
+
+The prometheus system is used for keeping track of the validator in a Kubernetes setting.
+However, this can also be used locally to run test case and then get metrics to work on.
+This is a little bit more indirect than having runtimes in log file, but works fine.
+The steps are the following:
+
+1. The list of created metric server can be seen in the validator logs with the entries "Starting to serve metrics".
```
Comment (ma2bd) [inline CONTRIBUTING.md:201]: `The list of the metric service ports` ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -191,6 +191,43 @@ A simple way to use this on a laptop is the following:
...
+2. The list of nodes served if 4 validors are used `is 0.0.0.0:XXXXX` with `XXXXX` being 11000, 11001, 11100, 11101, 11200, 11201, 11300, 11301.
+3. Create a `prometheus.yml` file that contains the source of data. Each one of the port has to be present. One example for querying every second is
```
Comment (ma2bd) [inline CONTRIBUTING.md:203]: typo: ports

File: CONTRIBUTING.md (hunk 2 above)
Comment (MathieuDutSik) [reply CONTRIBUTING.md:201]: Ok.

File: CONTRIBUTING.md (hunk 3 above)
Comment (MathieuDutSik) [reply CONTRIBUTING.md]: Ok, done.

**Pull request merged**